### PR TITLE
Fixed issue#3399

### LINKED ISSUE #3399: Feedback on page: /guides/v2.3/comp-mgr/cli/cli-rc1-samp.html
# General issue
There is a typo on the page, Magento Open Source mentioned twice by mistake.

## Description:

There is a typo on the page, Magento Open Source mentioned twice:
`These instructions apply to Magento Open Source (formerly Community Edition (CE) and Magento Open Source (formerly Enterprise Edition (EE) users only if all of the following are true`

## Possible solutions:

Change to: 
`These instructions apply to Magento Open Source (formerly Community Edition (CE) and Magento Commerce (formerly Enterprise Edition (EE) users only if all of the following are true`

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
Fixes #3399

When this pull request is merged, it will fix a typo.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: _includes/install/sampledata/sample-data-rc1-cli.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Command-line upgrade to Magento {{ page.guide_version }} with sample data
 
-These instructions apply to {{site.data.var.ce}} (formerly Community Edition (CE) and {{site.data.var.ce}} (formerly Enterprise Edition (EE) users *only* if all of the following are true:
+These instructions apply to {{site.data.var.ce}} and {{site.data.var.ce}} users *only* if all of the following are true:
```
Comment (jeff-matthews) [inline _includes/install/sampledata/sample-data-rc1-cli.md]:
```suggestion
These instructions apply to {{site.data.var.ce}} and {{site.data.var.ee}} users *only* if all of the following are true:
```

**Comment by gelanivishal** [comment]
@jeff-matthews Done.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**